File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/java/testcases/dataclasses/dataClassWithReservedWord/expected/DgsConstants.java
package com.netflix.graphql.dgs.codegen.java.testcases.dataclasses.dataClassWithReservedWord.expected;

import java.lang.String;

public class DgsConstants {
  public static final String QUERY_TYPE = "Query";

  public static class SAMPLETYPE {
    public static final String TYPE_NAME = "SampleType";

    public static final String Return = "return";
  }

  public static class QUERY {
    public static final String TYPE_NAME = "Query";

    public static final String People = "people";
  }

  public static class PERSON {
    public static final String TYPE_NAME = "Person";

    public static final String Info = "info";

    public static final String Interface = "interface";

    public static class INFO_INPUT_ARGUMENT {
      public static final String Package = "package";
    }
  }
}

File: graphql-dgs-codegen-core/src/integTest/kotlin/com/netflix/graphql/dgs/codegen/cases/dataClassWithReservedWord/expected/DgsConstants.kt
package com.netflix.graphql.dgs.codegen.cases.dataClassWithReservedWord.expected

import kotlin.String

@Generated
public object DgsConstants {
  public const val QUERY_TYPE: String = "Query"

  @Generated
  public object SAMPLETYPE {
    public const val TYPE_NAME: String = "SampleType"

    public const val Return: String = "return"
  }

  @Generated
  public object QUERY {
    public const val TYPE_NAME: String = "Query"

    public const val People: String = "people"
  }

  @Generated
  public object PERSON {
    public const val TYPE_NAME: String = "Person"

    public const val Info: String = "info"

    public const val Interface: String = "interface"

    @Generated
    public object INFO_INPUT_ARGUMENT {
      public const val Package: String = "package"
    }
  }
}
